lib/rules/no-invalid-regexp.js

```diff
@@ -148,11 +149,15 @@ module.exports = {
                 return "Regex 'u' and 'v' flags cannot be used together";
             }
 
+            if (duplicateFlags.length > 0) {
+                return `Duplicate flags supplied to RegExp constructor '${duplicateFlags.join("")}'`;
```

Comment:
Could we reformat this message? It reads a bit strange to me.
```suggestion
                return `Duplicate flags ('${duplicateFlags.join("")}') supplied to RegExp constructor.`;
```